vueapp/src/components/CreateGame.vue

```diff
@@ -8,29 +8,14 @@
     >
       <p class="text-h6">Select Players for Game</p>
       <v-list>
-        <v-list-item
-          v-for="player in playerStore.waitingPlayers"
-          :key="player.id"
-        >
-          <template v-slot:prepend>
-            <v-icon
-              :icon="
-                player.isGuest
-                  ? 'mdi-account-box-outline'
-                  : 'mdi-account-circle'
-              "
-            ></v-icon>
-          </template>
-          <v-list-item-title>
-            {{ player.name }}
-          </v-list-item-title>
-          <template v-slot:append>
+        <Player :player="player" v-for="player in playerStore.waitingPlayers">
```

Comment:
Don't forget the 'key' attribute to accompany the v-for to avoid the warning. Same in other places too.

Reply:
```suggestion
        <Player :player="player" v-for="player in playerStore.waitingPlayers" :key="player.id">
```